environments/docker.toml

```diff
@@ -11,22 +11,31 @@
         "flap_v1_0_9",
         "flip_bat_a_v1_0_0",
         "flip_bat_a_v1_0_9",
+        "flip_bat_a_v1_1_0",
         "flip_eth_a_v1_0_0",
         "flip_eth_a_v1_0_9",
+        "flip_eth_a_v1_1_0",
         "flip_knc_a_v1_0_8",
         "flip_knc_a_v1_0_9",
+        "flip_knc_a_v1_1_0",
         "flip_mana_a_v1_0_9",
+        "flip_mana_a_v1_1_0",
         "flip_sai_v1_0_0",
         "flip_tusd_a_v1_0_7",
         "flip_tusd_a_v1_0_9",
+        "flip_tusd_a_V1_1_0",
```

Comment:
super small, but I think we want this `V1` to be `v1` so that it matches with the exporter?

Reply:
Nice catch! 👍